pytest_xmlcomp.py

```diff
@@ -6,58 +6,108 @@
 from py.path import local
 import json
 from lxml import etree
+import pluggy
 
+hookspec = pluggy.HookspecMarker("pytest_xmlcomp")
+hookimpl = pluggy.HookimplMarker("pytest_xmlcomp")
```

Comment:
I guess, you don't need pluggy and the `hookspec` and `hookimpl` anymore. :wink: